app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,27 @@
+class SessionsController < ApplicationController
+
+	skip_before_action :logged_in?
+
+	def new
+		render '/sessions/new'
+	end
+
+	def create
+		@user = User.find_by(email: params[:email])
+
+		if @user && @user.authenticate(params[:password])
+			session[:user_id] = @user.id
+			redirect_to '/'
+		else
+			flash[:notice] = "Please Try Again"
+			render '/sessions/new'
```

Comment:
Try get in the habit of using helpers so redirect_to new_session_path - feels more railsy.